runtime/util/vmutil.tdf

```diff
@@ -58,4 +58,5 @@ TraceExit=Trc_VMUtil_getOriginalROMMethodUnchecked_Exit Overhead=1 Level=3 Noenv
 TraceException=Trc_VMUtil_getOriginalROMMethodUnchecked_getMethodIndexFailed Overhead=1 Level=1 Noenv Template="getOriginalROMMethodUnchecked: Unable to get methodIndex for method=%p"
 
 TraceEvent=Trc_Util_annhelp_SearchForFieldAnnotation Overhead=1 Level=5 Template="Search for field annotation: annotation=%.*s cpIndex=%zu clazz=%p romFieldShape=%p result=%zu"
+TraceEvent=Trc_Util_annhelp_SearchForMethodAnnotation Overhead=1 Level=5 Template="Search for method annotation: annotation=%.*s romMethod=%p result=%zu"
```

Comment:
@EricYangIBM @tajila New tracepoints must be added at the end (see https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md). I'll create a PR to fix this.

Reply:
See https://github.com/eclipse-openj9/openj9/pull/13184.